gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/remote/DriverRemoteConnection.java

```diff
@@ -101,6 +101,24 @@ private DriverRemoteConnection(final Cluster cluster, final boolean tryCloseClus
         this.conf = Optional.of(conf);
     }
 
+    /**
+     * Creates a {@link DriverRemoteConnection} using a new {@link Cluster} instance created from the supplied
```

Comment:
(nitpick) _from the supplied configuration file_ <- This overload takes no configuration file.
(Same for the other overload a few lines below.)